caproto/asyncio/utils.py

```diff
@@ -67,22 +67,20 @@ class _TransportWrapper:
 
     Parameters
     ----------
-    sock : socket.socket
-        The socket.
+    reader : ??
```

Comment:
I think these were the stream ones: https://docs.python.org/3/library/asyncio-stream.html#asyncio.StreamReader